### PR TITLE
enhancement: update block model

### PR DESCRIPTION
**Pull request by shiqizng**
Updating block model to include ParticipationUpdates.  cucumber test update [PR](https://github.com/algorand/algorand-sdk-testing/pull/234).

### CODE REVIEW

File: src/main/java/com/algorand/algosdk/v2/client/model/Block.java [hunk 1]
```diff
@@ -33,6 +33,12 @@ public String genesisHash() {
     @JsonProperty("genesis-id")
     public String genesisId;
 
+    /**
+     * Participation account data that needs to be checked/acted on by the network.
+     */
+    @JsonProperty("participation-updates")
+    public ParticipationUpdates participationUpdates;
```
Comment (jasonpaulos) [inline src/main/java/com/algorand/algosdk/v2/client/model/Block.java:40]: Are these code changes generated, or hand written?

**Review by jasonpaulos — COMMENTED** [review]
Seems to make sense, but I suppose we need to wait until https://github.com/algorand/algorand-sdk-testing/pull/234 is merged to have CI confirm

File: src/main/java/com/algorand/algosdk/v2/client/model/Block.java (hunk 1 above)
Comment (shiqizng) [reply src/main/java/com/algorand/algosdk/v2/client/model/Block.java:40]: generated.

**Comment by shiqizng** [comment]
sdk-testing updated

**Review by jasonpaulos — APPROVED** [review]
Thanks, looks good

**Review by winder — APPROVED** [review]

**Pull request closed without merging**